sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveFileFormat.scala

```diff
@@ -89,6 +90,8 @@ class HiveFileFormat(fileSinkConf: FileSinkDesc)
     val fileSinkConfSer = fileSinkConf
     new OutputWriterFactory {
       private val jobConf = new SerializableJobConf(new JobConf(conf))
+      private val broadcastHadoopConf = sparkSession.sparkContext.broadcast(
```

Comment:
Is it possible to use jobConf as hive serde initialize param directly?

Reply:
Thanks for the comment, I'll change code to use jobConf